openassessment/assessment/models/ai.py

```diff
@@ -40,6 +40,35 @@
 )
 
 
+def essay_text_from_submission(submission):
+    """
+    Retrieve the submission text.
+
+    Submissions are arbitrary JSON-blobs, which *should*
+    contain a single key, "answer", containing the essay
+    submission text.
+    If not, though, assume we've been given the essay text
+    directly (convenient for testing).
+    """
+    if isinstance(submission, dict):
```

Comment:
It seems a little odd to have the logic of a dict vs. unicode just for testing convenience. Is this really necessary?